src/pages/Home/Homepage.js

```diff
@@ -0,0 +1,193 @@
+import React from "react";
+import "./Homepage.css";
+
+const Homepage = () => {
+  return (
+    <>
+      <div className="homepage">
+        <div className="navbar-container">
+          <ul className="navbar nav-fixed d-flex">
+            <div className="d-flex">
+              <i className="fa fa-car hamberg-btn" aria-hidden="true"></i>
+              <a href="/index.html">
+                <p className="nav-title">Cars Store</p>
+              </a>
+            </div>
+            <div className="search-container d-flex">
+              <input
+                type="text"
+                name="search"
+                className="search-bar"
+                placeholder="Search for car"
+              />
+              <i className="fa fa-search fa-2x" aria-hidden="true"></i>
+            </div>
+            <ul className="navbar-right d-flex">
+              <li>
+                <a href="/cart/cart.html">
+                  <i className="fa fa-cart-plus fa-2x" aria-hidden="true"></i>
+                </a>
+              </li>
+              <li>
+                <a href="/wishlist/wishlist.html">
+                  <i className="fa fa-heart-o fa-2x" aria-hidden="true"></i>
+                </a>
+              </li>
+              <li>
+                <a href="/user profile/user_profile.html">
+                  <img
+                    src="https://picsum.photos/id/111/200/300"
+                    alt="navbar-img"
+                  />
+                </a>
+              </li>
+            </ul>
+          </ul>
+        </div>
+        <div className="homepage-container">
+          <div className="row">
+            <div className="header-column">
+              <h4>One stop solution for all vintage cars</h4>
+              <p>
```

Comment:
is it only for `Vintage cars` ? if yes then remove the categories below listed. else you need to change the sub-title to avoid ambiguity.

Reply:
Initially I thought to make it specific to the Vintage category but as I have changed the theme so I need to change the subtitle, this is for all kinds of cars and not just for Vintage cars.